ush/predef_grid_params.yaml

```diff
@@ -334,9 +401,6 @@
 #
 # The RRFS Alaska domain with ~3km cells.
 #
-# Note:
-# This grid has not been thoroughly tested (as of 20201027).
-#
```

Comment:
I'm assuming this shows up in the test suite now?

Reply:
Yes, the test is `grid_RRFS_AK_3km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR`